aries-site/src/pages/templates/ascending-navigation.mdx

```diff
@@ -0,0 +1,46 @@
+import { DiagramPreview, Example } from '../../layouts';
+
+When an application’s hierarchy is well designed, the ascending button will typically take the user 
+back to the previous page they came from.
+
+<Example 
+  caption="Example of an acending button." 
+  plain 
+  pad="small"
+  previewWidth="medium"
+  height={{ min: 'small' }}
+>
+  <DiagramPreview 
```

Comment:
Why is there a grey bar at the top of this image? It looks odd and at worst sloppy.

Reply:
I don't see the grey bar you're referring to...

Reply:
Ill fix this I had the wrong image